diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp
--- e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/120AppRegistryImp	2006-10-05 21:06:11.000000000 -0500
@@ -0,0 +1,9 @@
+// 120AppRegistryImp
+$this->applications['imp'] = array(
+    'fileroot' => dirname(__FILE__) . '/../imp',
+    'webroot' => $this->applications['horde']['webroot'] . '/imp',
+    'name' => _("Mail"),
+    'status' => 'active',
+    'provides' => 'mail',
+);
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders
--- e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/config/registry.php/170AppRegistryImpFolders	2006-10-05 21:06:54.000000000 -0500
@@ -0,0 +1,8 @@
+// 170AppRegistryImp
+$this->applications['imp-folders'] = array(
+    'status' => 'block',
+    'app' => 'imp',
+    'blockname' => 'tree_folders',
+    'menu_parent' => 'imp',
+);
+
