diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppImp mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppImp
--- e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppImp	1969-12-31 18:00:00.000000000 -0600
+++ mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/conf.menu.apps.php/02AppImp	2006-09-23 11:27:56.000000000 -0500
@@ -0,0 +1,5 @@
+{
+my $ImpMenu = $imp{MenuArray} || "enabled"; return "" unless ($ImpMenu eq "enabled");
+$apps{imp} = 1;
+$OUT = '';
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/conf.php/120MenuSettings mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/conf.php/120MenuSettings
--- e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/conf.php/120MenuSettings	2006-09-23 11:30:21.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.13.0/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/conf.php/120MenuSettings	2006-09-23 11:26:48.000000000 -0500
@@ -1,2 +1,2 @@
 // 120MenuSettings
-$conf['menu']['apps'] = array('turba');
+include '/home/httpd/html/horde/conf.menu.apps.php';
