diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035use_maillog mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035use_maillog
--- e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035use_maillog	2006-02-07 12:25:20.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035use_maillog	1969-12-31 19:00:00.000000000 -0500
@@ -1,4 +0,0 @@
-if (!empty($GLOBALS['conf']['maillog']['use_maillog'])) \{
-    $prefGroups['compose']['members'] = array_merge($prefGroups['compose']['members'], array('disposition_send_mdn'));
-\}
-
diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035viewing mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035viewing
--- e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035viewing	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/035viewing	2006-02-07 12:16:51.000000000 -0500
@@ -0,0 +1,10 @@
+$prefGroups['viewing'] = array(
+    'column' => _("Message Options"),
+    'label' => _("Message Viewing"),
+    'desc' => _("Configure how messages are displayed."),
+    'members' => array('filtering', 'strip_attachments',
+                       'html_image_replacement', 'html_image_addrbook',
+                       'highlight_text', 'highlight_simple_markup', 'show_quoteblocks',
+                       'dim_signature', 'emoticons', 'attachment_display')
+);
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040use_maillog mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040use_maillog
--- e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040use_maillog	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040use_maillog	2006-02-07 12:23:15.000000000 -0500
@@ -0,0 +1,4 @@
+if (!empty($GLOBALS['conf']['maillog']['use_maillog'])) \{
+    $prefGroups['viewing']['members'] = array_merge($prefGroups['compose']['members'], array('disposition_send_mdn'));
+\}
+
diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040viewing mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040viewing
--- e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040viewing	2006-02-07 12:25:20.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/040viewing	1969-12-31 19:00:00.000000000 -0500
@@ -1,10 +0,0 @@
-$prefGroups['viewing'] = array(
-    'column' => _("Message Options"),
-    'label' => _("Message Viewing"),
-    'desc' => _("Configure how messages are displayed."),
-    'members' => array('filtering', 'strip_attachments',
-                       'html_image_replacement', 'html_image_addrbook',
-                       'highlight_text', 'highlight_simple_markup', 'show_quoteblocks',
-                       'dim_signature', 'emoticons', 'attachment_display')
-);
-
diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/340disposition_send_mdn mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/340disposition_send_mdn
--- e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/340disposition_send_mdn	2006-02-07 12:25:20.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/prefs.php/340disposition_send_mdn	2006-02-07 12:22:08.000000000 -0500
@@ -3,7 +3,7 @@
     'locked' => false,
     'shared' => false,
     'type' => 'checkbox',
-    'desc' => _("Send MDN when requested by the sender?"),
+    'desc' => _("Send read receipt when requested by the sender?"),
     'help' => 'prefs-disposition_send_mdn'
 );
 
