diff -Nur -x '*.orig' -x '*.rej' e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/servers.php/120IMAPServers mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/servers.php/120IMAPServers
--- e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/servers.php/120IMAPServers	2006-01-20 10:26:21.000000000 -0500
+++ mezzanine_patched_e-smith-imp-1.11.2/root/etc/e-smith/templates/home/httpd/html/horde/imp/config/servers.php/120IMAPServers	2006-01-19 18:15:37.000000000 -0500
@@ -1,10 +1,12 @@
 {
     use esmith::DomainsDB;
     my $db = esmith::DomainsDB->open_ro;
-    foreach my $domain ($db->get_all())
+    my @domains = ($DomainName,
+	map { $_->key }
+	    grep { $_->prop('SystemPrimaryDomain') ne 'yes' }
+		$db->get_all());
+    foreach my $domain (@domains)
     {
-	$domain = $domain->key;
-
 	$OUT .= "\$servers['$domain'] = array(\n";
 	$OUT .= "    'name' => 'IMAP Server',\n";
 	$OUT .= "    'server' => 'localhost',\n";
